### PR TITLE
chore: update to more recent afgo version, go 1.16, and linter

### PR DESCRIPTION
**Pull request by Moopli**
Signed-off-by: Filip Burlacu <filip.burlacu@securekey.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -19,21 +19,21 @@ require (
...
 	github.com/stretchr/testify v1.7.0
-	github.com/trustbloc/edge-core v0.1.7-0.20210331113925-b13dedfe75eb
+	github.com/trustbloc/edge-core v0.1.7-0.20210426154540-f9c761ec6943
 	github.com/trustbloc/edv v0.1.7-0.20210420141323-971877b36098
 	github.com/trustbloc/kms v0.1.7-0.20210420100919-6aeea555441f
-	github.com/trustbloc/trustbloc-did-method v0.1.7-0.20210331120333-aacd187d2f94
+	github.com/trustbloc/trustbloc-did-method v0.1.7-0.20210430130101-b8e5fc60fb6d
 )
 
 replace github.com/kilic/bls12-381 => github.com/kilic/bls12-381 v0.0.0-20201104083100-a288617c07f1
```
Comment (troyronda) [inline go.mod:39]: What's with this replace?

Comment (Moopli) [reply go.mod:39]: Thanks for the catch, I don't think we need it anymore

**Review by rolsonquadras — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (Moopli) [reply go.mod:39]: same with the one in vc-rest

Comment (Moopli) [reply go.mod]: fixed

**Pull request merged**